staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -98,13 +102,26 @@ type EventList struct {
 // categories are logged.
 type Policy struct {
 	metav1.TypeMeta
+	// ObjectMeta is included for interoperability with API infrastructure.
+	// +optional
+	metav1.ObjectMeta
```

Comment:
> why? Which infrastructure?

Generation, generic clients, scope naming, API installer infrastructure (if we ever want it).  I've been down the path of excluding this before and you get whacky errors.  I don't see a reason to fight our infrastructure on a few fields, several of which we probably want in the end like annotations.